learning/views.py

```diff
@@ -1,9 +1,90 @@
-from django.shortcuts import render
+from django.shortcuts import render, HttpResponse, redirect
+from .question_functions import get_random_question_id
+
+# from django.views.decorators.http import require_http_methods, require_POST
+import json
+from .models import Question, Answers
+from .forms import QuestionForm
+import random
+from django.contrib.auth.decorators import login_required
 
 
 def index(request):
     user = None
     if request.user.is_authenticated:
         user = request.user
-    context = {"user": user}
+        questions = Question.objects.all()
+    next_word_id = get_random_question_id()
+    context = {"user": user, "question_id": next_word_id}
     return render(request, "index.html", context)
+
+
+@login_required(login_url="/login")
+def test(request, id):
```

Comment:
Ta funkcja test zostanie, czy miała być tymczasowa? Jeśli ma zostać, to lepiej, żeby nie było zahardcodowanych słów - też musimy przekmnić, jak dobierać złe słowa